MediaGalleryUi/Model/Directories/ExcludedDirectories.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\MediaGalleryUi\Model\Directories;
+
+/**
+ * Excluded directories validator
+ */
+class ExcludedDirectories
+{
+    /**
+     * @var array
+     */
+    private $excludedDirectories;
+
+    /**
+     * Constructor
+     *
+     * @param array $excludedDirectories
+     */
+    public function __construct(
+        array $excludedDirectories
```

Comment:
I think this name will better describe the content of the variable and property
```suggestion
        array $patterns
```